.travis.yml

```diff
@@ -23,7 +24,7 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset RAILS_ENV=test
-sudo: false
+sudo: required
```

Comment:
@jfly - it looks like MySQL has been [removed from trusty](https://github.com/travis-ci/docs-travis-ci-com/commit/d71108826c0de8387ce57c5c0519c7bce08e5cac). It looks like this means we'd need to use `sudo` to install it and hence couldn't use the container based infrastructure anyhow.